package.json

```diff
@@ -60,7 +60,7 @@
     "figlet": "^1.4.0",
     "file-loader": "^6.0.0",
     "html-webpack-plugin": "^5.5.0",
-    "inquirer": "^7.3.0",
+    "inquirer": "^9.3.0",
```

Comment:
@stefanhorning 
`inquirer` already is at v12, but it is not actively developed anymore. There is a new package replacing it - https://www.npmjs.com/package/@inquirer/prompts

I remember I tried latest version at some point but it had some issues... sadly don't remember what issues.

It would be really good to ditch packages that are not actively developed anymore. 
If you get blocked by something, please make tickets and I will get to it when I have time.
Other than this lgtm 👍

Reply:
actually had to rewrite some of my test-code to use that new module anyway as it was easier then to fix old code, so will actually use that now